src/citelao/Spotifious/Menus/DetailAlbum.php

```diff
@@ -25,13 +25,8 @@ public function __construct($options, $alfred, $api) {
 		$this->originalQuery = $options['originalQuery'];
 		$this->search = $options['search'];
 
-		if($api) {
-			$artistFetcher = new JsonFetcher("https://api.spotify.com/v1/albums/{$options['id']}");
-			$json = $artistFetcher->run();
-		} else {
-			$json = $api->getAlbum($options['id']);
-		}
-		
+		$json = $api->getAlbum($options['id']);
+
```

Comment:
This was actually a typo, and is why people were affected. You should use the API if you have it, not the other way round 😮